catkin_virtualenv/cmake/catkin_generate_virtualenv.cmake

```diff
@@ -112,14 +118,14 @@ function(catkin_generate_virtualenv)
   if(NOSETESTS)
     if(${ARG_PYTHON3})
       find_program(nosetests NAMES
-        "nosetests${PYTHON_VERSION_MAJOR}"
-        "nosetests-${PYTHON_VERSION_MAJOR}"
+        "nosetests3"
```

Comment:
I'm not totally sure about this. `nosetests` is either `nosetests` or `nosetests3`; it doesn't differentiate minor version.  Also `if(${ARG_PYTHON3})` leads me to believe that we can be explicit here instead of using the variable.

Reply:
This will be consistent for both pythons:

```
${CATKIN_ENV} ${PYTHON_EXECUTABLE} -m nose
```

Reply:
Thanks, copped this here https://github.com/locusrobotics/catkin_virtualenv/pull/33/files